Alcatraz/Installers/ATZPluginInstaller.m

```diff
@@ -65,10 +66,16 @@ - (NSString *)downloadRelativePath {
 // This is a temporary support for installs in /tmp.
 - (NSString *)pathForInstalledPackage:(ATZPackage *)package {
     NSString *pluginsInstallPath = [NSHomeDirectory() stringByAppendingPathComponent:INSTALLED_PLUGINS_RELATIVE_PATH];
-    NSString *pluginInstallName = [self installNameFromPbxproj:package] ?: package.name;
+    NSString *pluginInstallName = [self installNameFromPbxproj:package] ?: [package.name stringByAppendingString:package.extension];
 
-    return [[pluginsInstallPath stringByAppendingPathComponent:pluginInstallName]
-                                       stringByAppendingString:package.extension];
+    // Packages can be installed in multiple places:
+    // ~/Library/Application Support/Developer/Shared/Xcode/Plug-ins (where most xcplugins go)
+    // ~/Library/Developer/Xcode/Plug-ins (where ideplugins go)
```

Comment:
I don't follow what you mean by a predicate method in this case. Do you mean adding something like -(NSString*)installPath to ATZPlugin? I could do that, although that would mean stuff like installNameFromPbxproj would start leaking out of ATZPluginInstaller.

Should I remove those comments? I don't see how that's a comment detailing a change, it's saying what the code is actually doing. I'll remove it if that's the requested style.